common/services/base.test.js

```diff
@@ -1,77 +0,0 @@
-const proxyquire = require('proxyquire')
```

Comment:
Removing this keeps the revert commit simple, but I'd vote for this to be restored in the meantime, whatever the eventual resolution is.

Reply:
Not sure what you mean?